src/components/extensions/Errors.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { ERRORS } from "../../constant";
+import Table from "./Table";
+
+const Errors = ({ errors }) => {
+  const allErrors = errors.split(",");
+  const formattedErrors = allErrors.map((error) => {
+    const newError = ERRORS.find((item) => item.id === error);
+    return newError;
+  }).map((error) => {
+    const errorArray = [];
+    errorArray.push(<code className="language-text">{error.id}</code>)
+    errorArray.push(error.message)
+    errorArray.push(error.description)
+    return errorArray;
+  })
```

Comment:
This is good, but let's make it better to avoid double iteration :)

```suggestion
  const formattedErrors = allErrors.map((error) => {
    const newError = ERRORS.find((item) => item.id === error);
    
    return [
      <code className="language-text">{newError.id}</code>,
      newError.message,
      newError.description
    ];
  })
```